pkg/kritis/review/review_test.go

```diff
@@ -308,31 +289,23 @@ func TestReviewISP(t *testing.T) {
 	secFpr := sec.PgpKey.Fingerprint()
 	vulnImage := testutil.QualifiedImage
 	unQualifiedImage := "image:tag"
-	sigVuln, err := util.CreateAttestationSignature(vulnImage, sec)
+	attVuln, err := util.CreateAttestation(vulnImage, sec)
 	if err != nil {
 		t.Fatalf("unexpected error %v", err)
 	}
 
 	noVulnImage := testutil.IntTestImage
-	sigNoVuln, err := util.CreateAttestationSignature(noVulnImage, sec)
+	attNoVuln, err := util.CreateAttestation(noVulnImage, sec)
 	if err != nil {
 		t.Fatalf("unexpected error %v", err)
 	}
+
+	validAtts := []cryptolib.Attestation{*attVuln}
```

Comment:
This could use a comment. The attestation on a vulnerable image the valid one?

Reply:
Done. Renamed and added comment